ui/shared/components/table/RnaSeqJunctionOutliersTable.jsx

```diff
@@ -110,12 +110,12 @@ const RNA_SEQ_SPLICE_COLUMNS = [
 ]
 
 const RnaSeqJunctionOutliersTable = React.memo(
-  ({ variant, reads, showReads, updateReads, data, dispatch, ...props }) => (
+  ({ variant, reads, showReads, updateReads, data, columns, dispatch, ...props }) => (
     <div>
       {reads}
       <DataTable
         idField="idField"
-        columns={RNA_SEQ_SPLICE_COLUMNS}
+        columns={columns || RNA_SEQ_SPLICE_COLUMNS}
```

Comment:
I think it would be better to have a boolean field passed in and then select which set of columns to use based on that